packages/plugin/src/VisualizationPlugin.js

```diff
@@ -30,6 +30,7 @@ export const VisualizationPlugin = ({
     ...props
 }) => {
     const engine = useDataEngine()
+    const [ouLevels, setOuLevels] = useState(undefined)
```

Comment:
why explicitly set `undefined` instead of just `useState()`?

Reply:
I copied from v36 so no reason I guess 😁 
Fixed.